ci_framework/roles/repo_setup/tasks/rhos_release.yml

```diff
@@ -7,7 +7,9 @@
 
 - name: Get rhos-release tool version
   become: true
-  ansible.builtin.command: rhos-release --version
+  ci_script:
```

Comment:
same, not sure about the value to switch to ci_script

Reply:
Ack. Reverted.